greenfoot/src/main/java/greenfoot/sound/MicLevelGrabber.java

```diff
@@ -48,7 +49,9 @@ private MicLevelGrabber()
         format = new AudioFormat(22050, 8, 1, true, true);
         updator = () -> {
             try {
-                TargetDataLine line = (TargetDataLine) AudioSystem.getLine(new DataLine.Info(TargetDataLine.class, format));
+                Mixer mixer = SoundUtils.loadMixer(true);
```

Comment:
I suppose a small explanation as to the difference of having a mixer available and using it makes in this portion of code can be added here